frontend/src/app/components/docs/api-docs-nav.component.html

```diff
@@ -0,0 +1,63 @@
+<p *ngIf="network.val !== 'bisq' && network.val !== 'liquid'">General</p>
+<a *ngIf="network.val !== 'bisq' && network.val !== 'liquid'" [routerLink]="['./']" fragment="get-difficulty-adjustment" (click)="collapseItem.toggle()">GET Difficulty Adjustment</a>
+
+<p *ngIf="network.val === 'bisq'">Markets</p>
```

Comment:
Consider replacing whole blocks of identical *ngIf rows with a single wrapper like:
```
<ng-template [ngIf]="network.val === 'bisq'">
   <a> ... </a>
   <a> ... </a>
   <a> ... </a>
</ng-template> 
```